demos/src/Nodes/Mention/React/suggestion.js

```diff
@@ -60,6 +66,10 @@ export default {
       onUpdate(props) {
         component.updateProps(props)
 
+        if (!props.clientRect) {
+          return
+        }
```

Comment:
While this is nice, I don't think this actually works.

Down below, you see:

`getReferenceClientRect: props.clientRect`

When you look at Tippy's typings, it goes something like this:

```ts
export interface GetReferenceClientRect {
  (): ClientRect | DOMRect;
  contextElement?: Element;
}

...
export interface Props extends LifecycleHooks, RenderProps {
  getReferenceClientRect: null | GetReferenceClientRect;
  ...
}
```

Tippy could handle if `props.clientRect` is null, but in practice I've seen it be the function. Tippy expects `() => ClientRect | DOMRect` whereas we've got `() => ClientRect | DOMRect | null`.

It is kinda problematic.

On the one hand, you could run the function yourself, see if it is `null` and then pass it in if it is not, but that doesn't really work because it could be `null` by the time Tippy runs it.

Reply:
I see. Tricky issue since I wasn't able to reproduce it even by blowing up my machine with 10k console.logs per editor state change. 🤡 

I'll think about this a bit and search for a better and cleaner solution.

The worst solution but also the worst case could be that we pass out a fake clientRect object but that feels pretty messy.